docs/Changelog.md

```diff
@@ -18939,6 +18939,49 @@ This version of the operator has been available since version 14 of the default
 <dd>Constrain seq_lens to integer tensor.</dd>
 </dl>
 
+### <a name="GridSampler-14"></a>**GridSampler-14**</a>
+
+  Given an input and a flow-field grid, computes the output using input values and pixel locations from grid.
+  Currently, only spatial (4-D) input are supported.
+
+#### Version
+
+This version of the operator has been available since version 14 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>align_corners</tt> : int (default is 0)</dt>
+<dd>If align_corners=1, the extrema (-1 and 1) are considered as referring to the center points of the input's corner pixels. If align_corners=0, they are instead considered as referring to the corner points of the input's corner pixels, making the sampling more resolution agnostic.</dd>
```

Comment:
the align_corners option selected here should be in sync\same as with the option selected for resizing the image (if necessary)  before grid sampling right?

Reply:
Yes, I think that means a grid generated by affine_grid() should be passed to grid_sample() with the same setting for this option.
refer link: https://pytorch.org/docs/master/generated/torch.nn.functional.affine_grid.html?highlight=align_corners.